pom.xml

```diff
@@ -60,4 +47,16 @@
         </license>
     </licenses>
 
+    <modules>
+        <module>api</module>
+        <module>spec</module>
+        <module>impl</module>
```

Comment:
The `jdsol-spec` modules should probably also be on this module list

Reply:
I could add that in another PR. This PR is mostly about the formatting and plug-ins etc.